conformance/README.md

```diff
@@ -40,15 +40,43 @@ tests using similar patterns.  You can either use Bazel to run the
 or create an executable for a custom test and pass it to
 `conformance_test_runner`.
 
-For example, to run the Ruby tests against MRI, you can call:
-
-    $ bazel test //ruby:conformance_test
-
 Note: CMake can be used to build the conformance test runner, but not any of
 the conformance test executables outside C++.  So if you aren't using Bazel
 you'll need to create the executable you pass to `conformance_test_runner` via
 some alternate build system.
 
+While we plan to model all our supported languages more completely in Bazel,
+today some of them are a bit tricky to run.  Below is a list of the commands
+(and prerequisites) to run each language's conformance tests.
+
+Java:
+    $ bazel test //java/core:conformance_test //java/lite:conformance_test
```

Comment:
Add a link break between, and surround the command with backticks to get the formatting to match the previous section. Apply to all of the languages.

Reply:
The backtick doesn't seem to do anything for the formatting, but I'll add the line break and try to make these more copy-pastable

Reply:
Intro text on one line, a blank line, a line with three backticks, the lines with code, a line with three backticks, and then another blank line should work. This is according to https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#fenced-code-blocks

Reply:
Ah gotcha, this isn't really code though.  It's commands to be copy/pasted.  The way it is now seems more consistent with our other command snippets, how's this look?

Reply:
For some background: Sandy was having a really hard time figuring out how to actually run these conformance tests, some of which are still in a very intermediate state.  So I just wanted to have something to point people at before I forget the magic commands :)

Reply:
LGTM